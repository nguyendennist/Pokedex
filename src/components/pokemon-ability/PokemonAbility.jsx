import React from 'react';
import PropTypes from 'prop-types';

const PokemonAbility = ({ability}) => (
	<p>{ability.name}</p>
);

PokemonAbility.propTypes = {
	ability: PropTypes.object.isRequired
};

export default PokemonAbility;
