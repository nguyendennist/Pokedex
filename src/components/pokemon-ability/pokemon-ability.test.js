import React from 'react';
import { create } from 'react-test-renderer';
import PokemonAbility from './PokemonAbility.jsx';

describe('unit tests', () => {
  test('should be defined', () => {
    expect(PokemonAbility).toBeDefined();
  });
});

describe('ui tests', () => {
  test('should render correctly', () => {
    const tree = create(
      <PokemonAbility ability={{}} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
