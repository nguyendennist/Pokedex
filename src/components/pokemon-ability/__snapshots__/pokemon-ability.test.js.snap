// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui tests should render correctly 1`] = `<p />`;
